introduction_to_amazon_algorithms/autogluon_tabular/Amazon_Tabular_Regression_AutoGluon.ipynb

```diff
@@ -0,0 +1,564 @@
+{
```

Comment:
<p>Style nit: I think there's no need to have <code>ABALONE</code> in all caps -&gt; <code>Abalone</code>  <span class="ql-cursor">﻿</span></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3461/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>Thanks! I have changed them in all regression notebook</p>